ocs_ci/ocs/constants.py

```diff
@@ -527,7 +528,8 @@
 
 # Rep mapping
 REPO_MAPPING = {
-    '4.2.0': OCP4_2_REPO
+    '4.2.0': OCP4_2_REPO,
```

Comment:
Cannot we fix with the same approach you did in AWS?

Reply:
Should be doable, this was just a quick fix.

Reply:
We can go by this quick fix by now and fix properly in other PR